docs/products/postgresql/reference/pg-connection-limits.md

```diff
@@ -9,13 +9,17 @@ follows:
 
 | Plan                                                    | Max Connections |
 | ------------------------------------------------------- | --------------- |
+| Free (DigitalOcean only)                                | 20              |
 | Hobbyist (Google Cloud, DigitalOcean, and UpCloud only) | 25              |
 | Startup/Business/Premium-4                              | 100             |
 | Startup/Business/Premium-8                              | 200             |
 | Startup/Business/Premium-16                             | 400             |
 | Startup/Business/Premium-32                             | 800             |
 | Startup/Business/Premium-64 and above                   | 1000            |
 
+Three connections are reserved for the platform for administrative purposes,
```

Comment:
I recommend not to add this explicit information at this point.

Reply:
That's fair - it might change, after all. Perhaps a more general statement could be considered for the future? The context is that [a user was unable to connect](https://aiven.io/community/forum/t/suddently-my-app-cant-connect-with-my-db/1152), and was uncertain as to how many connections were available for use. It might not be obvious that not all listed connections will be available for use by an app, so I was trying to explain possible reasons.

Reply:
@GreenReaper thanks for much for your contribution :-) I addressed your comment via https://github.com/aiven/aiven-docs/pull/268